CoreFoundation/String.subproj/CFRegularExpression.h

```diff
@@ -57,6 +57,7 @@ _CFRegularExpressionRef _Nullable _CFRegularExpressionCreate(CFAllocatorRef allo
 void _CFRegularExpressionDestroy(_CFRegularExpressionRef regex);
 
 CFIndex _CFRegularExpressionGetNumberOfCaptureGroups(_CFRegularExpressionRef regex);
+CFIndex _CFRegularExpressionGetCaptureGroupNumberWithName(_CFRegularExpressionRef regex, CFStringRef groupName) API_AVAILABLE(macosx(10.13), ios(11.0), watchos(4.0), tvos(11.0));
```

Comment:
Why availability annotations? Internal calls do not need them, and in this case they would also be inaccurate (this call isn’t on any of the mentioned OSes.)

Just remove them, here and in the Swift code?